charts/trino/values.yaml

```diff
@@ -418,8 +427,7 @@ env: []
 # ```
 
 envFrom: []
-# envFrom -- additional environment variables added to every pod, specified as a list of either `ConfigMap`
-# or `Secret` references
+# envFrom -- additional environment variables added to every pod, specified as a list of either `ConfigMap` or `Secret` references
```

Comment:
Revert this change

Reply:
Hmm, I initially missed `Secret` as they are in different lines. So I thought it does not support `Secret`. I saw other comments is much wider than this line, so I guess fine (?)

Reply:
It's fine, but it's an unnecessary change.